release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -190,6 +190,15 @@ For more information, see xref:../networking/external_dns_operator/nw-installing
 
 SR-IOV support is now available for xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Pensando DSC cards]. OpenShift SR-IOV is supported, but you must set a static, Virtual Function (VF) media access control (MAC) address using the SR-IOV CNI config file when using SR-IOV.
 
+[id="ocp-4-11-support-for-hosted-control-planes"]
+==== Hosted control planes (Technology preview)
+
+In {product-title} 4.11, a technology preview is available for hosted control planes. Hosted control planes for {product-title} is a new deployment model that removes the requirement that clusters be self-hosted. Instead, a hosted cluster's control plane exists as pods on another management cluster.
+
+Adapting the OVN networking plug-in to the hosted control planes model requires careful design. OVN provides its own control plane and data store, all of which are external to the hosted cluster.
```

Comment:
@lahinson, do we describe how to do this elsewhere? Is that in the upstream HyperShift docs repo?

Reply:
@jboxman-rh I'm not sure if this is documented elsewhere, but I plan to find out.

Reply:
I think we can trim the "careful" bit. Since OVN will be the default in hosted control-planes, how should a user be careful, can we remove it? 

"OVN provides its own control plane and data store, with hosted control planes, OVN was re-design to host its control-plane and datastore alongside the cluster's control-plane .... " Please paraphrase as you see fit :)